### ISSUE TITLE
gen: `+vats, =verb |` not printing correctly

### ISSUE THREAD

**Issue by tinnus-napbus** [urbit/urbit#6476]
**Describe the bug**

```
> +vats, =verb |
 %groups /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414] app status:       running publishing ship:  [~ ~sogryp-dister-dozzod-dozzod] pending updates:  ~::
 %base /sys/kelvin:      [%zuse 414] app status:       running publishing ship:  ~ pending updates:  ~::
 %landscape /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414] app status:       running publishing ship:  [~ ~lander-dister-dozzod-dozzod] pending updates:  ~::
 %webterm /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414] app status:       running publishing ship:  [~ ~mister-dister-dozzod-dozzod] pending updates:  ~::
 %garden /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414] app status:       running publishing ship:  [~ ~mister-dister-dozzod-dozzod] pending updates:  ~::
 %kids %kids %cz hash:     0v13.gb7b2.v1h24.upgv3.o743l.jjc2k.8ellh.8keur.4osot.1si86.rll9k::
 %talk /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414] app status:       running publishing ship:  [~ ~sogryp-dister-dozzod-dozzod] pending updates:  ~::
 ```

**To Reproduce**
Steps to reproduce the behaviour:
1. `+vats, =verb |`
2. prints ugly

**Expected behaviour**
should not print ugly

**System (please supply the following information, if relevant):**
 - OS: linux64
 - Vere 2.1 & latest develop

**Notify maintainers**

@hoclun-rigsep

**Comment by tinnus-napbus** [comment]
wait it's working correctly for me now despite not changing anything... weird. Maybe my terminal was just being weird. I'll reopen if it recurs

**Comment by tinnus-napbus** [comment]
wait no it's still messed up sometimes and in some places it seems:

```

  %gol-cli
  /sys/kelvin:      [%zuse 417] [%zuse 416]
  app status:       suspended
  publishing ship:  ~
  pending updates:  ~[[%zuse 416]]
::
  %portal
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  app status:       running
  publishing ship:  [~ ~worpet-bildet]
  pending updates:  ~
::
 %channel /sys/kelvin:      [%zuse 418] app status:       suspended publishing ship:  ~ pending updates:  ~::
  %silo
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  app status:       running
  publishing ship:  ~
  pending updates:  ~
::
 %kids %kids %cz hash:     0v8.n40h1.hjn8c.e762q.ncgh0.e4gq3.6n3l5.5kt8l.6gtdr.ovbah.u3avd::
 %cli /sys/kelvin:      [%zuse 414] app status:       suspended publishing ship:  ~ pending updates:  ~::
 %oger /sys/kelvin:      [%zuse 418] app status:       suspended publishing ship:  ~ pending updates:  ~::
  %tally
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  app status:       running
  publishing ship:  ~
  pending updates:  ~
::
  %escape
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415]
  app status:       suspended
  publishing ship:  ~
  pending updates:  ~
::
  %talk
  /sys/kelvin:      [%zuse 417] [%zuse 416] [%zuse 415] [%zuse 414]
  app status:       running
  publishing ship:  [~ ~sogryp-dister-dozzod-dozzod]
  pending updates:  ~
::
```

**Comment by joemfb** [comment]
Fixed in #6532.
